lang/nl.js

```diff
@@ -20,7 +20,7 @@ export default {
 	"components.filter.clearDescription": "Filters wissen voor {filterName}",
 	"components.filter.loading": "Laden van filters",
 	"components.filter.filterCountDescription": "{number, plural, =0 {Geen filters toegepast.} one {{number} filter toegepast.} other {{number} filters toegepast.}}",
-	"components.filter.filters": "Filters", // mfv-translated
+	"components.filter.filters": "filters",
```

Comment:
This one's interesting - capitalization has been changed, so the `// mfv-translated` is no longer needed. I'm not sure whether this was a deliberate change or not...

Reply:
Hmm. The English one is uppercase, as are the others it seems... I feel like it wasn't deliberate then?

Reply:
Yeah I thought maybe it was a title-case difference, but "Filters" is used on its own, and our other NL one-word text is capitalized. I'll restore it to the original.